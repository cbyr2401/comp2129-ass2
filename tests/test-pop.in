POP a
POP b
POP c
SET a 1 2 3
LIST ENTRIES
POP a
LIST ENTRIES
POP a
LIST ENTRIES
POP a
LIST ENTRIES
POP a
LIST ENTRIES
BYE