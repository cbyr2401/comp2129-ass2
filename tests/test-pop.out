> no such key

> no such key

> no such key

> ok

> a [1 2 3]

> 1

> a [2 3]

> 2

> a [3]

> 3

> a []

> nil

> a []

> bye
